package.json

```diff
@@ -36,10 +36,10 @@
     "root": true
   },
   "dependencies": {
-    "@enact/core": "^3.2.0",
-    "@enact/i18n": "^3.2.0",
-    "@enact/spotlight": "^3.2.0",
-    "@enact/ui": "^3.2.0",
+    "@enact/core": "^3.3.0-alpha.8",
```

Comment:
please remove these changes as they are part of PR FORTECH-7

Reply:
revert npm-shrinkwrap.json also

Reply:
done